integration-tests/app-test/src/app.tsx

```diff
@@ -2,17 +2,32 @@ import { useState } from 'preact/hooks'
 import preactLogo from './assets/preact.svg'
 import viteLogo from '/vite.svg'
 import './app.css'
-// import { Metrics } from '@edgio/rum'
+import { Metrics, trackConversion } from '@edgio/rum'
 import Routes from './views/routes'
 
-//added RUM
-// const RUM_TOKEN = "cea882df-d1bb-4547-8dce-5d0fc9a89d2b"
-// new Metrics({
-//   token: RUM_TOKEN,
-// }).collect()
+ const RUM_TOKEN = "cea882df-d1bb-4547-8dce-5d0fc9a89d2b"
+ const metrics = new Metrics({
+   token: RUM_TOKEN,
+   pageLabel: "my-label-page",
+   appVersion: 'v1.0.0',
+   cacheHit: true,
+   country: 'US'
+})
+metrics.collect()
```

Comment:
Let's be careful with these changes. I am really not sure if these changes are correct by quick glance but check this out. 
https://github.com/layer0-platform/rum/blob/main/integration-tests/app-test/package.json#L11

The idea is that we have the same app and we just swap the way we install the RUM.

Reply:
Hm, but what is the difference? It seems they are the same now, I added the same code in all templates files.

Reply:
You are right, it looks like the setup is correct right now 👍